### PR TITLE
create sub function

### PR DESCRIPTION
**Pull request by hupaulcamacho**
@aransagarcia @lakerfan1994 @SavitaMadray

### CODE REVIEW

File: sub.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const sub = (num1, num2) => {
+    return num1 - num2
+};
+
+module.exports = sub;
```
Comment (aransagarcia) [inline sub.js:5]: LOOKS GOOD!

Comment (MambaStacks) [inline sub.js:5]: I am writing this here because I am writing this to fit in the comment character count.

**Review by MambaStacks — APPROVED** [review]
Good code

File: sub.js (hunk 1 above)
Comment (hupaulcamacho) [reply sub.js:5]: thanks

**Review by aransagarcia — APPROVED** [review]

**Pull request merged**